internal/input/ledgers.go

```diff
@@ -5,30 +5,30 @@ import (
 
 	"github.com/stellar/stellar-etl/internal/utils"
 
-	"github.com/stellar/go/xdr"
+	"github.com/stellar/go/historyarchive"
 )
 
 // GetLedgers returns a slice of ledger close metas for the ledgers in the provided range (inclusive on both ends)
-func GetLedgers(start, end uint32, limit int64, isTest bool, isFuturenet bool) ([]xdr.LedgerCloseMeta, error) {
+func GetLedgers(start, end uint32, limit int64, isTest bool, isFuturenet bool) ([]historyarchive.Ledger, error) {
 	env := utils.GetEnvironmentDetails(isTest, isFuturenet)
 	backend, err := utils.CreateBackend(start, end, env.ArchiveURLs)
 	if err != nil {
-		return []xdr.LedgerCloseMeta{}, err
+		return []historyarchive.Ledger{}, err
 	}
 
-	metaSlice := []xdr.LedgerCloseMeta{}
-	ctx := context.Background()
+	ledgerSlice := []historyarchive.Ledger{}
 	for seq := start; seq <= end; seq++ {
-		ledger, err := backend.GetLedger(ctx, seq)
+		ctx := context.Background()
```

Comment:
nit: does ctx need to be inside the for loop?

Reply:
moving it out